spark/src/main/scala/org/apache/spark/sql/delta/storage/dv/DeletionVectorStore.scala

```diff
@@ -91,6 +91,10 @@ trait DeletionVectorStoreUtils {
   /** The size of the stored length of a DV. */
   final val DATA_SIZE_LEN = 4
 
+  def getTotalSizeOfDVFieldsInFile(bitmapDataSize: Int): Int = {
```

Comment:
I don't see this method used anywhere. Can we update the existing places where the size calculated is used with this method call?

Also the method name should be? `getTotalSizeOfDVInFile`?

Reply:
updated the usage in the Suite. Seems to be the only reference. 

And that name is suggested by @larsk-db

Reply:
Yeah, please keep that name. It actually says what it really does. `getTotalSizeOfDVInFile` is confusing, because really that is the number you are passing in as `bitmapDataSize`. The rest of the size is meta-information fields.